app/controllers/course_summaries_controller.rb

```diff
@@ -79,7 +79,9 @@ def download_csv_grades_report
     grades_data = get_table_json_data(current_role)
 
     csv_string = MarkusCsv.generate(grades_data, [generate_csv_header, generate_out_of_row]) do |student|
-      row = [student[:user_name], student[:first_name], student[:last_name], student[:id_number]]
+      row = Student::CSV_ORDER.map do |field|
```

Comment:
So the `do ... end` is the syntax for a multi-line block. However, there's a more concise syntax for *single-line blocks*, which looks like `{ |field| student[field] }`. Try using this syntax throughout.